examples/point-of-sale/package.json

```diff
@@ -20,47 +20,46 @@
     },
     "dependencies": {
         "@solana/pay": "^0.2.4",
-        "@solana/spl-token": "^0.3.5",
-        "@solana/wallet-adapter-base": "^0.9.16",
-        "@solana/wallet-adapter-react": "^0.15.18",
-        "@solana/wallet-adapter-react-ui": "^0.9.16",
-        "@solana/wallet-adapter-wallets": "^0.18.9",
-        "@solana/web3.js": "^1.56.0",
-        "bignumber.js": "^9.1.0",
-        "bs58": "^4.0.1",
-        "cache-manager": "^4.1.0",
+        "@solana/spl-token": "^0.3.6",
+        "@solana/wallet-adapter-base": "^0.9.19",
+        "@solana/wallet-adapter-react": "^0.15.25",
+        "@solana/wallet-adapter-react-ui": "^0.9.23",
+        "@solana/wallet-adapter-wallets": "^0.19.7",
+        "@solana/web3.js": "^1.69.0",
+        "bignumber.js": "^9.1.1",
+        "bs58": "^5.0.0",
```

Comment:
I don't think this should be changed, web3.js still uses v4 AFAIK.

Reply:
bs58 v5 was release 10 months ago !!
I tested it and it's working fine with web3.js.